.werft/util/certs.ts

```diff
@@ -35,30 +43,29 @@ export async function issueCertficate(werft, params: IssueCertificateParams) {
         throw new Error(`there is no subdomain + '${params.domain}' shorter or equal to 63 characters, max. allowed length for CN. No HTTPS certs for you! Consider using a short branch name...`);
     }
 
-
     // Always use 'terraform apply' to make sure the certificate is present and up-to-date
     var cmd = `set -x \
+    && export KUBECONFIG="${params.pathToKubeConfig}" \
     && cd ${params.pathToTerraform} \
+    && rm -rf .terraform* \
```

Comment:
We reuse the same dir to create certs for both dev cluster and k3s cluster. Therefore, we remove any residue tf from previous execution to avoid getting terraform warnings.

Reply:
Yes. And that is fine, as we can retrieve it any time using terraform. This step is only to make sure that subsequent k3s ws cluster cert creation doesn't get impacted